build/Version.props

```diff
@@ -1,8 +1,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <VersionMajor>2</VersionMajor>
-    <VersionMinor>2</VersionMinor>
-    <VersionPatch>0</VersionPatch>
+    <VersionMinor>1</VersionMinor>
+    <VersionPatch>300</VersionPatch>
     <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">preview1</ReleaseSuffix>
```

Comment:
Are we going to change the master suffix to preview2?

Reply:
Yes. Pr coming